pkg/virt-launcher/virtwrap/access-credentials/access_credentials.go

```diff
@@ -285,16 +285,14 @@ func (l *AccessCredentialManager) agentSetFilePermissions(domName string, filePa
 }
 
 func (l *AccessCredentialManager) agentSetUserPassword(domName string, user string, password string) error {
-
-	base64Str := base64.StdEncoding.EncodeToString([]byte(password))
-
-	cmdSetPassword := fmt.Sprintf(`{"execute":"guest-set-user-password", "arguments": {"username":"%s", "password": "%s", "crypted": false }}`, user, base64Str)
-
-	_, err := l.virConn.QemuAgentCommand(cmdSetPassword, domName)
+	domain, err := l.virConn.LookupDomainByName(domName)
```

Comment:
The domain needs to be freed, because it is created in C code and it's not managed by golang's garbage collector.